multimodal/src/autogluon/multimodal/utils/export.py

```diff
@@ -73,82 +76,71 @@ def dump_model(self, save_path: Optional[str] = None):
 
     def export_onnx(
         self,
-        data: Optional[pd.DataFrame] = None,
```

Comment:
We may need to improve this example and add more examples about `timm_image`, `hf_text`, and `fusion_mlp`. Since `data` is required, I guess this example no longer works. You can mark this add a TODO item. I think tutorials may also be necessary to teach users.

Reply:
We might need to consider clean up these examples, since they are not tested / verified in contineous-integration. See https://github.com/autogluon/autogluon/issues/2848 

> I think tutorials may also be necessary to teach users.

I'm planning to add a tutorial for using TRT, but the tutorial would come after this PR.

Reply:
A tutorial will be added in https://github.com/autogluon/autogluon/pull/2987